File: MVVMJsonExample/app/src/main/java/com/blackbirds/shakil/mvvmjsonexample/network/APIService.java
package com.blackbirds.shakil.mvvmjsonexample.network;

import com.blackbirds.shakil.mvvmjsonexample.model.MovieModel;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface APIService {
    @GET("marvel")
    Call<List<MovieModel>> getMovieList();
}

File: KotlinMVVMJsonExample/app/src/main/java/com/blackbirds/shakil/kotlinmvvmjsonexample/Network/APIService.kt
package com.blackbirds.shakil.kotlinmvvmjsonexample.Network

import com.blackbirds.shakil.kotlinmvvmjsonexample.Model.MovieModel
import retrofit2.Call
import retrofit2.http.GET

interface APIService {
    @GET("marvel")
    fun getMovieList(): Call<MutableList<MovieModel>>
}